src/expressvpn/index.html

```diff
@@ -41,7 +41,7 @@ <h3><i>Scraper-UI.</i></h3>
                             <img alt="expressvpn"  src="https://upload.wikimedia.org/wikipedia/en/thumb/7/79/ExpressVPN-logo.svg/261px-ExpressVPN-logo.svg.png?20210118095637" />
                         </div>
                         <div class="box2">
-                            <a href="#" target="_blank">
+                            <a href="https://www.expressvpn.com/jobs/job-openings" target="_blank">
                             Careers
                             </a>
                             <a href="#" target="_blank">
```

Comment:
you forgot targetare.ro link
```suggestion
                            <a href="targetare.ro_link" target="_blank">
```

Reply:
Got it.